SPEC.md

```diff
@@ -77,7 +77,7 @@ An `issue` represents a single instance of a real or potential code problem, det
 * `content` -- **Optional**. A markdown snippet describing the issue, including deeper explanations and links to other resources.
 * `categories` -- **Required**. At least one category indicating the nature of the issue being reported.
 * `location` -- **Required**. A `Location` object representing the place in the source code where the issue was discovered.
-* `other_locations` -- **Optional.** An array of `Location` objects useful for engines which highlight more than one source location in an issue.
+* `trace` -- **Optional.** A `Trace` object representing other interesting source code locations related to this issue.
```

Comment:
Do we have engines built on top of `other_locations` yet? Do we need to flesh out versioning and compatability concerns before this can move forward?